### PR TITLE
a few more diagnostics changes

### PR DESCRIPTION
**Pull request by StephenHodgson**
Overview
---
Found a missing namespace reference (Current dev branch doesn't compile)
Fixed profile locking.

- Fixes: #2864

### CODE REVIEW

File: Assets/MixedRealityToolkit/_Core/Inspectors/Profiles/MixedRealityDiagnosticsSystemProfileInspector.cs [hunk 1]
```diff
@@ -42,6 +43,11 @@ public override void OnInspectorGUI()
                 Selection.activeObject = MixedRealityManager.Instance.ActiveProfile;
             }
 
+            if (MixedRealityPreferences.LockProfiles && !((BaseMixedRealityProfile)target).IsCustomProfile)
```
Comment (keveleigh) [inline Assets/MixedRealityToolkit/_Core/Inspectors/Profiles/MixedRealityDiagnosticsSystemProfileInspector.cs]: Should this go in the `MixedRealityBaseConfigurationProfileInspector` so we don't have to add it to each individual inspector?

Comment (keveleigh) [reply Assets/MixedRealityToolkit/_Core/Inspectors/Profiles/MixedRealityDiagnosticsSystemProfileInspector.cs]: Ah, I see what you mean about the profile button. If there's a way to do it in the base, that'd be preferable, but it works as-is and that can be addressed later (and called out in the documentation).

Comment (StephenHodgson) [reply Assets/MixedRealityToolkit/_Core/Inspectors/Profiles/MixedRealityDiagnosticsSystemProfileInspector.cs]: I tried to put it in the base initially, but it's not trivial.

**Comment by SimonDarksideJ** [comment]
Resolves #2863

**Pull request merged**